app/views/devise/confirmations/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Resend confirmation instructions</h2>
+
+<%= form_for(resource, as: resource_name, url: confirmation_path(resource_name), html: { method: :post }) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", value: (resource.pending_reconfirmation? ? resource.unconfirmed_email : resource.email) %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Resend confirmation instructions" %>
+  </div>
+<% end %>
+
```

Comment:
- [ ] Please improve the login page design in order to match the design given 😃 👍 See the comparison below 👇 
**Given design**
![image](https://user-images.githubusercontent.com/90222110/209351393-1927abea-56fc-41db-99dd-de714f180103.png)

**Your design**
![image](https://user-images.githubusercontent.com/90222110/209351543-c6bd4301-a521-4aa7-b1ee-97909242b51e.png)

Please make sure you stick to the design given 👍